src/SFA.DAS.ApplyService.Domain/Apply/GatewayApplicationSummaryItem.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace SFA.DAS.ApplyService.Domain.Apply
+{
+    public class GatewayApplicationSummaryItem
```

Comment:
This looks to be the same information displayed across Gateway, FHA and Assessor views, so wondered if this could become more generic?

Reply:
Make this generic so we now have RoatpApplicationSummaryItem